draft-erdtman-jose-cleartext-jws.xml

```diff
@@ -407,15 +408,110 @@
         signatures but must also take the following in account as well:
         <list style="symbols">
           <t>
-            The <spanx style="verb">__cleartext_signatures</spanx> property must contain an array
+            There MUST be an additional Cleartext JWS Header parameter <spanx style="verb">signers</spanx>, holding an array
             of signature constructs.
           </t>
           <t>
             Each signature requires its own normalization process. During this
-            process, the other signature objects must (temporarily) be removed.
+            process, the other signature constructs MUST (temporarily) be removed.
           </t>
-        </list>
+          <t>
+            The normalized data MUST include the array brackets ([]) but MUST NOT
+            include the comma separating the signature constructs.
+          </t>
+          <t>
+            A signature construct is equivalent to an ordinary Cleartext JWS object,
+            but MAY exclude the <spanx style="verb">alg</spanx> Header parameter which then
```

Comment:
Is it only alg and crit that may be put at global level? if there are more I would like us to call out the rule that headers must only exist at one level.

Reply:
I haven't found any other header variable that would be meaningful to share.